playground/backend/internal/cloud_bucket/precompiled_objects.go

```diff
@@ -52,6 +52,7 @@ type ObjectInfo struct {
 	Categories      []string                 `json:"categories,omitempty"`
 	PipelineOptions string                   `protobuf:"bytes,3,opt,name=pipeline_options,proto3" json:"pipeline_options,omitempty"`
 	Link            string                   `protobuf:"bytes,3,opt,name=link,proto3" json:"link,omitempty"`
+	Graph           string                   `protobuf:"bytes,7,opt,name=graph,proto3" json:"graph,omitempty"`
```

Comment:
I think we need to store graph separately in a file like graph.dot, not in meta.info

Reply:
Based on our discussion we send string representation of the graph to the frontend, not a file

Reply:
Changed.

Reply:
Shouldn't we remove this from `ObjectInfo` then?